modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -17,15 +17,16 @@ Existing control plane `BareMetalHost` objects may have the `externallyProvision
 
 .Procedure
 
-. Remove the old `baremetalhost`, `machine` and `node` objects:
+. Remove the old `baremetalhost` and `machine` objects:
```

Comment:
In the important admonition at the start of this module, the "`BareMetalHost` object" is referred to. You could either change your "`baremetal` object" references throughout this PR to use the same capitalisation as in the admonition, or vice-versa. I recommend reviewing which of the two is the correct capitalisation for the object name. My guess is that it's "`BareMetalHost`", based on the YAML in step 2.

Reply:
@pneedle-rh You're right and it seems that `machine` should be `Machine` as well. Updated accordingly

Reply:
Thank you for highlighting that. Please also review if `node` should be `Node` when referring to the object.

Reply:
Looking through the docs library, the `Node` object is capitalised too.

Reply:
Thank you, Paul!